ui/src/components/SampleView/SampleView.css

```diff
@@ -89,14 +89,6 @@
   position: relative;
 }
 
-.btn:focus {
-  outline: none !important;
-}
-
-.btn:active {
-  color: grey !important;
```

Comment:
This style would apply when pressing (and remaining pressed) on any button in the UI. It was quite ugly in my opinion, as it would make button labels/icons unreadable on click. I couldn't find a button where this style made the active state look better, so I think it's safe to remove it. If there's a regression somewhere, it should be barely noticeable. But do let me know if you notice one.